test/selenium-webdriver/search_screencaps.py

```diff
@@ -0,0 +1,58 @@
+import os
+import requests as rs
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.common.by import By
+
+
+SCREEN_DIR = './screenshots'
+TARGET_URL = 'https://search.creativecommons.org'
+release = rs.get('https://api-dev.creativecommons.engineering/version').json()
```

Comment:
This is temporarily pointing at api-dev until the next production release of cccatalog-api.